content/docs/ops/maintenance-list.md

```diff
@@ -29,7 +29,7 @@ The tasks on this checklist should be performed each day.
 
 ## Review and respond to open alerts
 
-Review all recent alerts and notifications delivered to `cg-notifications` and `#cloud-gov-atlas-news`.  
+Review all recent alerts and notifications delivered to `cg-notifications` and [`#cg-platform-news`](https://gsa-tts.slack.com/messages/cg-platform-news/).
```

Comment:
hmm, I don't see that channel anywhere @brittag 

<img width="477" alt="screen shot 2017-05-12 at 12 50 22 pm" src="https://cloud.githubusercontent.com/assets/706004/26008164/9f6f4e58-3711-11e7-9959-ac0f70c56035.png">